### ISSUE TITLE
Categorize settings

### ISSUE THREAD

**Issue by nanab** [Manuel83/craftbeerpi3#30]
As there going to be lots ofta plugins whit its own settings.
There would be great if we could add a catergory or similar to the settings for cleaner listing.

**Comment by Manuel83** [comment]
Good idea. To be honest I don't expected that after one week are already 10 plugins available.

**Comment by jalim** [comment]
I was just thinking that some sort of category system might be needed for the plugin page as well?  At this rate its going to get rather hard for new users to wade through all the plugins!
